backend/Dockerfile

```diff
@@ -22,7 +22,7 @@ ARG POETRY_EXTRA_ARGS=''
 RUN poetry export --without-hashes --format=requirements.txt ${POETRY_EXTRA_ARGS} && \
     pip install -r requirements.txt
 
-COPY setup.py README.md ./
+COPY setup.py ./
```

Comment:
I needs ``` README.md `` https://github.com/inspirehep/inspirehep/blob/master/setup.py#L14

Reply:
why?

Reply:
https://github.com/inspirehep/inspirehep/blob/master/setup.py#L14

Reply:
if we remove it from Dockerfile we should remove it from the setup.py

Reply:
your links are for the current master, it's already in the diff.

Reply:
my bad, haven't seen the changes in the ``setup.py``